### ISSUE TITLE
Training for content editors for alt text improvements

### ISSUE THREAD

**Issue by BerniXiongA6** [department-of-veterans-affairs/va.gov-cms#15549]
## Problem Statement
CMS team needs to provide updated training for content editors as we implement the alt text improvements.

## What you need to know
- This ticket should be completed prior to the launch of the alt text validation improvements
- There's an existing KB article for alt text information for editors (could potentially be something we edit for training)
- There's an existing alt text page on the VA design system (which is content focused)
- FYI for Product: This work must be completed before Product can create Rollout strategy and plan (BX to include ticket once that's created to show dependencies)

## Acceptance Criteria
- [x] Updated article for how to create good alt text (why it's important)
- [x] Article is shared with Product for review/feedback
- [x] Discussion of where to put the new article (Content style guided, KB, somewhere else?)
- [x] [Change management statement](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15944) is drafted

**Comment by BerniXiongA6** [comment]
Hey @BlakeOrgan -- ERika and I created this ticket for upcoming sprint work, please let us know if you have anything to add to the details or AC ahead of refinement. Thanks! cc: @EWashb

**Comment by BerniXiongA6** [comment]
Hey @BlakeOrgan and @laflannery let's try to refine before Sprint Planning, I can either do it async or we can hop on a call ahead of sprint planning. Lmk thanks!

**Comment by BerniXiongA6** [comment]
Hi @laflannery @BlakeOrgan -- if schedules sync up better later today, we can meet post-sprint-planning too! Thanks!

**Comment by BerniXiongA6** [comment]
Note for @EWashb -- while you were gone, we changed this ticket. There's an associated ticket that also achieves our primary goal for creating this work -- see #15655 cc: @BlakeOrgan @laflannery

**Comment by laflannery** [comment]
I just realized that if editors go to edit an existing image, maybe they want to add a new crop ratio, and their alt text has any of these errors (for example if it is over 150 characters) they are going to get flagged. They aren't going to be able to save their image again without fixing their alt text even if that wasn't the purpose of why they were editing their image in the first place. This is a use case we should account for in the change management/training

**Comment by EWashb** [comment]
Oh good point @laflannery. We should think this through because that would be a big change to the workflow for them. I would say that use case increases the CM tshirt size

**Comment by BerniXiongA6** [comment]
@laflannery if you find that S96 has you spread thin, we can still have you partially complete this ticket in S96 and carry over to S97 if need be. I'm OK with that given we only allot 2pts per sprint for a11y work!

**Comment by laflannery** [comment]
[New/Updated article for alt text guidance](https://docs.google.com/document/d/1_V2JfLMBnY6hSTyk5M6MS0lnOqV6cO9Hc8FCSu9buLE/edit#heading=h.2g3fvchjm8bh)

Here's why thought - we have [an existing article](https://design.va.gov/content-style-guide/alternative-text-for-images) in the Content style guidelines. However, this doesn't necessary explain WHY we are asking editors to follow these guidelines and from the research Jo did there was definitely some confusion around that (Example: at least one editor felt strongly that that uses would want to know that it was a photo, not knowing the screen reader already indicates this).

So I wanted to write a training article to better explain/show why these guidelines are in place and what the experience might be like if they aren't followed.

The question now is - where do we put this?
- The KB has historically been for just "How to Drupal" articles
- The Design System site has been just for content guidelines

This is an article that doesn't seem to fit in either place. @EWashb was going to reach out to Randi H and see if she had opinions.

**Comment by laflannery** [comment]
I forgot to add this to be sure folks saw it - I used the Change management ticket template to create [this plan/statement](https://github.com/department-of-veterans-affairs/va.gov-cms/issues/15944) which might help to think through the communications that need to happen

**Comment by EWashb** [comment]
I love that additional ticket. Super thorough. I want to make sure we plan out time to execute the CM plan BEFORE launching (obviously)

**Comment by laflannery** [comment]
I've thought about this a bit more and actually talked a bit about it with Jo and I think it does make more sense to have this in the KB (rather than try to get this to fit into the DS website somehow). We do want the KB to evolve and be a more robust training space and I think this type of articles does fall into this type of more extensive training where editors can get a more complete understanding of what they are being asked to do.

Unless there are any objections (from Randi or others) I would like to find a place for this within the KB @EWashb @BerniXiongA6 thoughts?

**Comment by EWashb** [comment]
That works for me @laflannery

**Comment by maortiz-27-80** [comment]
@EWashb - are we good to publish the updated article in google doc from Laura? [New/Updated article for alt text guidance](https://docs.google.com/document/d/1_V2JfLMBnY6hSTyk5M6MS0lnOqV6cO9Hc8FCSu9buLE/edit#heading=h.2g3fvchjm8bh)

The issue is that everything at the top of the the page until **Why does following these tips lead to good alt text?**
 is duplicative content that already exists in the DS.  Are we okay with having duplicative content or should we ask Randi for guidance?  cc: @laflannery @BerniXiongA6

**Comment by EWashb** [comment]
@maortiz-27-80 I cannot see the doc you linked since I'm on the VPN. Do you have it somewhere else or in another format?

**Comment by maortiz-27-80** [comment]
@EWashb attached as a PDF cc: @laflannery @BerniXiongA6 
[Alt text Validation change management.pdf](https://github.com/department-of-veterans-affairs/va.gov-cms/files/13835135/Alt.text.Validation.change.management.pdf)

**Comment by EWashb** [comment]
Looks good to me @maortiz-27-80

**Comment by maortiz-27-80** [comment]
Thanks, @EWashb !  @laflannery - I've created https://github.com/department-of-veterans-affairs/va.gov-cms/issues/16685.  cc: @BerniXiongA6
